rfcs/text/0000-simple-updates.md

```diff
@@ -0,0 +1,161 @@
+
+For these reasons, the DCP must support additions, updates, deletions, and retractions of data and metadata in submitted projects including: adding a publication 1 year after a dataset was submitted, updating descriptive fields due to improvements in understanding and ontological modeling of anatomy or cellular identity, fixing metadata errors or relationships between entities, removing unconsented data.
+
+This RFC proposes a simple technical design to support the simplest metadata updates and additions, recognizing that this is the first step towards a complete solution.
+
+### Definitions
+
+***Addition*** - adding a metadata field:value pair to a metadata document where the metadata field existed in the schema version described by the metadata document  
+***Update*** - changing the value of an existing metadata field in a metadata document
+***Old update process*** - the current update process which involves adding the existing project to an exclusion list and re-ingesting the entire project - with updates - from scratch as a new submission  
+***Data wrangler*** - member of the DCP team who interacts with data contributors to collect metadata updates and additions and drives the process of submitting them to the DCP  
+***Experimental design representation*** - the connected set of entities which the DCP uses to represent the experimental design of a project  
+***Bundles*** - The data store representation of a logical set of data and metadata. There are two types of bundles input and analysis. Input bundles represent a single set of experimental data such as a lane or plex of sequencing and are defined by ingest as a biomaterial to file transition, connected by an assay process and all entities upwards in the experimental design from that point. Analysis bundles are all the analysis results from a single run of an analysis pipeline on a single input bundle and are defined by ingest as a file to file transition as defined by an analysis process and all entities upwards in the experimental design from that point.
+
+## User Stories
+
+As a data contributor, I would like to add the details of a scientific publication to my submission to the DCP to broaden knowledge of the work.
+
+As a data contributor, I would like to correct an error I have made in my submission to ensure that the correct metadata is presented to consumers of my data.
+
+As a data wrangler, I would like to update organ system metadata to reflect an improved ontology annotation for this particular organ.  
+
+Read more on the general user experience background for this work: [User Experience Problem Space - Top task “Update my project”](https://docs.google.com/document/d/1FP6kJa3f9NkOHTLqmkNYztls7soZLgeGefxdgR81ONE/edit)
+
```

Comment:
In the _user experience background_ material reference above, this is commented as MVP:
 
> Edit the data file names for existing files

Could you offer an example -and- is this in scope for this RFC?

Reply:
@gabsie is the owner of that document and I am uncertain where the requirement for file name changes came from, hopefully, she can comment.

As far as MVP goes, these other documents made statements about what might be considered MVP but this RFC reflects the scope for the current work which is restricted based on capacity vs other documents.

Reply:
@brianraymor @lauraclarke @justincc 
Hey. So I put this in my initial problem space doc, since I wanted to address the scenario of making a human error in the names of things. This is just one example: putting the wrong filename in the sheet with filenames and needing to correct it.

If this is not covered by the current simple update scenario, then I am OK to edit things in my doc re: priority. I can definitely see how initial submitters or even data wranglers can make human errors when pasting/spelling/putting the wrong filename across the sample/etc. I have been a witness of such in a couple of discussions between wrangler and submitter.

Let me know what to do next, my document is quite high level and deals with general user needs, which are very true, but which might not necessarily be addressed in this first phase.

Reply:
No further action is required for the RFC from my perspective since Laura confirmed that the scope is restricted based on the RFC and not the reference material. I simply wanted to confirm.